README.md

```diff
@@ -23,8 +23,8 @@ The ACME provider that most people are likely to use is [Let's Encrypt](https://
 
 ```
 > $ docker run --rm praekeltfoundation/marathon-acme:develop marathon-acme --help
-usage: marathon-acme [-h] [-a ACME] [-m MARATHON] [-l LB [LB ...]] [-g GROUP]
-                     [--listen LISTEN]
+usage: marathon-acme [-h] [-a ACME] [-m MARATHON [MARATHON ...]]
+                     [-l LB [LB ...]] [-g GROUP] [--listen LISTEN]
```

Comment:
I'm not a huge fan of space-separated opts like `-m foo bar baz`, because it looks like the extras are positional args.

Can we make these comma-separated or something?